### ISSUE TITLE
Update README

### ISSUE THREAD

**Issue by cassgvp** [alan-turing-institute/tric-dt#17]
The intention is for this repo to be relevant to all work of the TRIC-DT, and not just the Hub. As such the README should reflect all TRIC-DT interest/activities, but managed by the Hub. 

The readme has been copied from a TRIC-DT-Hub repo. This needs to be reviewed for appropriate content and updated to contain all relevant community information. Suggestions for inclusion here on [Open WIN](https://open.win.ox.ac.uk/pages/open-science/community/Open-WIN-Community/docs/gitlab/repo-include/#readmemd) (via OLS).

**Comment by cassgvp** [comment]
Hi @ZGGhauch !

I've just updated the README with some basic information 🥳 . Would love to hear your comments on what useful information we could include in there! Lots of the detail is missing, but your early thoughts would be great!

Feel free to comment on this issue or submit a PR with any suggestions for changes! Thank you!

**Comment by cassgvp** [comment]
@aranas Would you like to add something about your seminar series to the REAME? Perhaps in the CONTRIBUTING section?

**Comment by kallewesterling** [comment]
This issue is too broadly formulated to ever get closed, and as such, needs to be split up into smaller, resolvable issues. Alternatively, we can keep this issue open as a meta-issue linking to the other issues, so we can track progress for updates on the README file.

**Comment by chrisdburr** [comment]
> This issue is too broadly formulated to ever get closed, and as such, needs to be split up into smaller, resolvable issues. Alternatively, we can keep this issue open as a meta-issue linking to the other issues, so we can track progress for updates on the README file.

Agreed. @cassgvp can you update the README to a place you're happy with, add the changes here and then we can close this issue. Any specific future updates can then have new issues.

**Comment by cassgvp** [comment]
Only thing missing is a link to the knowledge commons jupyter book (currently links to #6). This issue can be closed after that has been added!
